WS2012R2/lisa/setupscripts/ChangeCPU.ps1

```diff
@@ -105,11 +151,11 @@ if ($procs)
 {
     if ($procs -is [array])
     {
-        $maxCPUs = $procs[0].NumberOfCores
+        $maxCPUs = $procs[0].NumberOfLogicalProcessors *2
```

Comment:
Switching to NumberOfLogicalProcessors makes sense, but why is it multiplied by 2? The physical CPUs x2 give the logical cores count.